src/proposal.ts

```diff
@@ -154,7 +153,7 @@ export type ParamsCronInfo = {
 };
 
 export type ParamsDexInfo = {
-  fee_tiers: number[];
+  fee_tiers: (string | number)[];
```

Comment:
why is it `string | number` now? was it a deliberate change or a misprint?

Reply:
ah, so in our integration tests there is this check, so i made it so it doesn't error:
```
      // types mixed on purpose, to check contract parser.
      // Numeric types in neutron-std can be deserialized from both number and string
      fee_tiers: ['1', '2', 99],
```